src/services/webview/WebViewMessageRoutingService.ts

```
@@ -0,0 +1,543 @@
+
+      case 'splitTerminal':
+        log('🔀 [DEBUG] Splitting terminal from webview...');
+        context.splitTerminal();
+        break;
+
+      case 'focusTerminal':
+        log('🎯 [DEBUG] ========== FOCUS TERMINAL COMMAND RECEIVED ==========');
+        const terminalId = message.terminalId as string;
+        
+        const currentActive = context.terminalManager.getActiveTerminalId();
+        log(`🔍 [DEBUG] Current active terminal: ${currentActive}`);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)